test/test_sparse.py

```diff
@@ -1135,7 +1135,7 @@ def test_shape(di, dj, dk, nnz):
         "bmm sparse-dense CUDA is not yet supported in Windows, at least up to CUDA 10.1"
     )
     @unittest.skipIf(
-        TEST_CUDA and _get_torch_cuda_version() < (10, 1),
+        TEST_CUDA and _get_torch_cuda_version() < (10, 1) and not TEST_WITH_ROCM,
```

Comment:
Are we wanting this test to be skipped or unskipped?

TEST_CUDA will always be True.

_get_torch_cuda_version() returns (0,0) if torch.version.cuda is None, so that check will be True.

TEST_WITH_ROCM is our env var, which will be True for us.

So this test needs to be skipped?

Reply:
not TEST_WITH_ROCM, which turns it into false and it doesn't skip.